README.md

```diff
@@ -1,2 +1,71 @@
 # gobra
 Gobra is a graphical user interface (GUI) for the cobra command line interface (cli).
+
+# Usage
+
+Gobra generates an HTML snippet.
+
+```go
+import (
+	"[thisrepo]"
```

Comment:
What is this for?

Reply:
When I was cloning your repo to mine, I ran into trouble because even though it's cloned to `github.com/khoin/gobra` the code still imports from `ctessum/gobra`, so I looked it up and people advised me to still clone from your repo, but set `git remote` to my repo so when I push changes I push to my repo instead. 

I'll change `[thisrepo]` to `github.com/ctessum/gobra` later.